doc/README.md

```diff
@@ -31,6 +31,7 @@ Reference Documentation
 * [Router radixtree](router-radixtree.md)
 * [Stand Alone Model](stand-alone.md): Supports to load route rules from local yaml file, it is more friendly such as under the kubernetes(k8s).
 * [Stream Proxy](stream-proxy.md)
+* [gRPC Proxy](grpc-proxy.md)
```

Comment:
Do we need to update the Chinese version?

Reply:
Addressed this in latest commit.